schedule/qam/common/qem-live-migration.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: live_migration
+description:    >
+    scheduling tests of live migration
+schedule:
+   - '{{host_install}}'
+   - '{{migrate_src}}'
+   - '{{migrate_dst}}'
+conditional_schedule:
+   host_install:
+      HOST_INSTALL:
+         1:
+            - autoyast/prepare_profile
+            - boot/boot_from_pxe
+            - autoyast/installation
+            - virt_autotest/login_console
+   migrate_src:
+      MIGRATION_ROLE:
+         src:
+            - virtualization/livemigration/live_migration_src
+   migrate_dst:
+      MIGRATION_ROLE:
+         dst:
+            - virtualization/livemigration/live_migration_dst
+
+test_data:
+   location:
+      "sles-15": "http://dist.suse.de/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/"
```

Comment:
should be http://dist.suse.de/install/SLP/SLE-15-Installer-GM/x86_64/

Reply:
Hi Gou,
Thanks for review.  
I am planing to remove these data and add one or two lines code to generate them dynamically as they have some pattern.  
I am just curious why GM, not LATEST?  They both work as boot source.

Reply:
IMHO, LATEST link should be the soft link from GM one - GM is the real existing dir sharing for us. We'd better use the GM for the released SLE version. continue to use LATEST for SLE Dev.

Reply:
ok, will use GM.